tools/buildsys/src/args.rs

```diff
@@ -57,6 +58,9 @@ pub(crate) struct Common {
     #[arg(long, env = "BUILDSYS_ARCH")]
     pub(crate) arch: SupportedArch,
 
+    #[arg(long, env = "BUILDSYS_CARGO_METADATA_PATH")]
```

Comment:
does this need to be added to the rebuild vars?

Reply:
> does this need to be added to the rebuild vars?

I think we're covered by the fact that the package/variant dirs are being watched for changes, and the cargo metadata output should be the same unless something has changed about those packages. In other words, I think if anything important changes, we will be rebuilding anyway.